.gitignore

```diff
@@ -4,6 +4,8 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
 
+echo '.env*' >> .gitignore
```

Comment:
Should that line be here? I think it's the terminal command to append '.env*' to the .gitignore file, no? (already specified in line 12-14)